app/utils/keymaps/normalize.js

```diff
@@ -0,0 +1,7 @@
+module.exports = keybinding => {
```

Comment:
Would it make sense to add a comment describing why this is needed and what its meant to do? It's not super-clear from reading this code 👍  would make it great for new contributors.

Reply:
Nice idea! Thanks for the tip. :)

Reply:
Great 👌